cl/corpus_importer/utils.py

```diff
@@ -315,23 +315,51 @@ def clean_docket_number(docket_number: str) -> str:
     :param docket_number: Case docket number
     :return: A stripped down docket number.
     """
-
+    docket_number = docket_number.strip(".")
     docket_number = re.sub("Department.*", "", docket_number)
-    docket_number = re.sub("Nos?. ", "", docket_number)
+    # Remove No. or Nos.
+    docket_number = re.sub("Nos?.? ", "", docket_number)
+    # remove word Docket
+    docket_number = re.sub("Docket ", "", docket_number)
+    # Remove Crim. in Crim. 40801
+    docket_number = re.sub("Crim.? ", "", docket_number)
+    # Remove Civ. in Civ. 7394
+    docket_number = re.sub("Civ.? ", "", docket_number)
+    # remove Petition Nos. in Petition No.
+    docket_number = re.sub("Petition Nos?.? ", "", docket_number)
+    # remove Record No. in Record No. 3331
+    docket_number = re.sub("Record Nos?.? ", "", docket_number)
+    # remove File in File CV-03 0402585
+    docket_number = re.sub("File ", "", docket_number)
+    # remove judgement affirmed in "31130. Judgment affirmed."
+    docket_number = re.sub("Judgment affirmed.?", "", docket_number)
+    # remove cause transferred in "31342. Cause transferred."
+    docket_number = re.sub("Cause transferred.?", "", docket_number)
+    # remove parentheses/brackets in (AC 16691)
+    docket_number = re.sub("[()]", "", docket_number)
+    # remove comma in No. 28,049
+    docket_number = docket_number.replace(",", "")
```

Comment:
it may be useful to reduce these to numbers for comparison sake but we dont want to remove strings from docket numbers in all cases

Reply:
No problem, i can leave that in another separate function, since it could be useful to us in the future